src/ansys/pytwin/evaluate/tbrom.py

```diff
@@ -120,10 +118,11 @@ def generate_snapshot(self, on_disk: bool, output_file_path: str, named_selectio
             vec = vec + mc[i] * mnp
         if named_selection is not None:
             pointsids = self.named_selection_indexes(named_selection)
-            listids = []
-            for i in pointsids:
-                for k in range(0, self.field_output_dim):
-                    listids.append(i * self.field_output_dim + k)
+            if self.field_output_dim > 1:
```

Comment:
I think this might fail for scalar outputs (`dimensionality = 1`), since `listids` is then not defined

Reply:
good catch Ed, thanks